dali/python/nvidia/dali/experimental/dynamic/_op_builder.py

```diff
@@ -415,13 +504,20 @@ def build_fn_wrapper(op):
     fn_name = _to_snake_case(op.schema.OperatorName())
     inputs = _get_inputs(schema)
 
+    has_random_state = schema.HasArgument("_random_state") and schema.IsTensorArgument(
+        "_random_state"
+    )
```

Comment:
I think it deserves a function in OpSchema (at least in the PyBind wrapper for it). Another option - just set it to `False` here and then handle it in the loop below.
```suggestion
    has_random_state = False
```